twint/url.py

```diff
@@ -68,17 +68,20 @@ async def Profile(username, init):
     return url
 
 async def Search(config, init):
-    logme.debug(__name__+':Search')
-    url = f"{base}/search/timeline"
+    logme.debug(__name__ + ':Search')
+    url = "https://mobile.twitter.com/search"
     q = ""
     params = [
         ('vertical', 'default'),
         ('src', 'unkn'),
         ('include_available_features', '1'),
         ('include_entities', '1'),
-        ('max_position', str(init)),
+        #('max_position', str(init)),
         ('reset_error_state', 'false'),
+        ('prefetchTimestamp', str(int(time.time() * 1000)))
     ]
+    if init != '-1':
+        url += init
     if not config.Popular_tweets:
         params.append(('f', 'tweets'))
```

Comment:
I think this needs to be `('f', 'live')`, as that is what happens in the URL when the "Latest" tab is selected.
 `https://mobile.twitter.com/search?q=potato&src=recent_search_click&f=live`